core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1347,24 +1347,9 @@ paths:
       summary: Paginated query for entities in a workspace
       operationId: entityQuery
```

Comment:
Originally I wanted pull the .yaml from https://rawls.dsde-dev.broadinstitute.org/api-docs.yaml and just paste that in here but the diff was massive since it changes a lot of ordering. So instead of having a computer merge these error-free, I manually merged them. I've poked around a bit and it looked good to me but this swagger change is worth giving some 👀